framework/src/outputs/Console.C

```diff
@@ -613,6 +609,11 @@ Console::outputScalarVariables()
 void
 Console::outputSystemInformation()
 {
+  // skip system information output for sub-apps other than the zero-th of a MultiApp
+  // because they are using the same inputs and are most likely having the same information.
+  if (_app.multiAppNumber() > 0)
+    return;
+
```

Comment:
Why we skip this? If we do not skip, will we have too many messages ?

Reply:
Yes, too many especially with hundreds of subapps.

Reply:
Yeah that is a little obnoxious

Reply:
does this stop the output of convergence history? I would like to be able to see all subapp solves in some instances when debugging.

Reply:
This is for reducing this output on screen:
```
pebble000: Parallelism:
pebble000:   Num Processors:          1
pebble000:   Num Threads:             1
pebble000: 
pebble000: Mesh: 
pebble000:   Parallel Type:           replicated
pebble000:   Mesh Dimension:          1
pebble000:   Spatial Dimension:       1
pebble000:   Nodes:                   30
pebble000:   Elems:                   28
pebble000:   Num Subdomains:          8
pebble000: 
pebble000: Nonlinear System:
pebble000:   Num DOFs:                30
pebble000:   Num Local DOFs:          30
pebble000:   Variables:               "T_pebble" "T_triso" 
pebble000:   Finite Element Types:    "LAGRANGE" "LAGRANGE" 
pebble000:   Approximation Orders:    "FIRST" "FIRST" 
pebble000: 
pebble000: Auxiliary System:
pebble000:   Num DOFs:                28
pebble000:   Num Local DOFs:          28
pebble000:   Variables:               "k_s" 
pebble000:   Finite Element Types:    "MONOMIAL" 
pebble000:   Approximation Orders:    "CONSTANT" 
pebble000: 
pebble000: Execution Information:
pebble000:   Executioner:             Transient
pebble000:   TimeStepper:             IterationAdaptiveDT
pebble000:   Solver Mode:             Preconditioned JFNK
pebble000:   PETSc Preconditioner:    hypre boomeramg 
pebble000: 
```
It does not affect the output of convergence history.

Reply:
ok nice

Reply:
I agree with @fdkong - I definitely don't want this skipped.  It can be turned off using the boolean parameter though.  But should default to on.